Makefile

```diff
@@ -213,6 +213,8 @@ failpoint-disable: tools/bin/failpoint-ctl
 	@$(FAILPOINT_DISABLE)
 
 tools/bin/ut: tools/check/ut.go
+	sudo  yum install -y centos-release-scl; \
+	sudo yum install -y devtoolset-8-libasan-devel; \
```

Comment:
I suggest removing this. Not every one is using yum.

Reply:
Now we can install those in our docker image. so it is useless.